tests/readrows_test.go

```diff
@@ -387,3 +387,63 @@ func TestReadRows_NoRetry_EmptyTableNoRows(t *testing.T) {
 	res := doReadRowsOp(t, server, &req, nil)
 	assert.Len(t, res.Row, 0)
 }
+
+// TestReadRows_NoRetry_MultipleRowRanges tests that the client can request multiple
+// row ranges to scan
+func TestReadRows_NoRetry_MultipleRowRanges(t *testing.T) {
+	k1 := "abar"
+	k2 := "kbar"
+	k3 := "qbar"
+	k4 := "zbar"
+
+	// 1. Instantiate the mock server
+	rec := make(chan *readRowsReqRecord, 3)
+	seq := []*readRowsAction{
+		{
```

Comment:
We don't need `&readRowsAction` here? Similar question for &btpb.RowRange? 

If so, that's great.

Reply:
Yeah! If you've already declared a variable as a slice of `readRowsAction` (or whatever), you don't need to instantiate elements of the slice with a type declaration.

Reply:
Good to know, thanks!